29.task/29.task.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<title>Document</title>
+</head>
+<body>
+	<script>
+		// if you want to get new random image you must reload your page
+		const img = document.createElement('img')
+		let i = Math.floor(Math.random() * (1 + 8) +1)
```

Comment:
```
const i = Math.floor(Math.random() * 9 + 1);
```

Reply:
really !

Reply:
it's so simplify  !

Reply:
Imagine, the backend has changed the names of images, and there are no numbers in the images names

More flexible solution
```
const images = ['./images/1.jpg', './images/2.jpg', .....];

img.src = images[Math.floor(Math.random() * images.length +1)];
```